server/src/api/posts/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const Posts = require('../../db/Posts');
+
+const posts = express.Router();
```

Comment:
could you rename this to
```suggestion
const postsRouter = express.Router();
```